net/tac_plus/patches/020-gcc14.patch

```diff
@@ -0,0 +1,30 @@
+--- a/tacacs-F4.0.4.28/do_author.c
++++ b/tacacs-F4.0.4.28/do_author.c
+@@ -30,6 +30,8 @@
+ # endif
+ #endif
+ 
++int cfg_get_logauthor(void);
```

Comment:
This declaration would perhaps be better placed on **tac_plus.h:421**

Reply:
Looks like it compiles there.